nodes/test/unit/test_model.py

```diff
@@ -38,7 +27,7 @@
 PARENT_SETTINGS = {constants.NODE_CERTIFICATE: NODE_CERTIFICATE}
 
 
-class TestModel(WebTest):
+class TestModel(ServerTests):
```

Comment:
I think this is the right change for now, but generally I think we should get rid of these intermediary test running helper classes that provide side effects. I envision the future of Pulp to inherit directly from unittest.TestCase directly.